### DISCUSSION TITLE
Using proguard rules from aar & maven dependencies

### DISCUSSION THREAD

**Discussion by jewelshkjony** [shreyashsaitwal/rush-cli#46] [Ideas]
Hello dear, I don't know that this feature is already available or not in Rush 2.0.

If not then please add this feature, it'll save our time to setting proguard rules for every dependencies.

**Comment by Ashishthakoor** [comment]
Yes, I also agree with this point.

**Comment by shreyashsaitwal** [comment]
Will be available in the next release. 👍🏻
